README.md

```diff
@@ -75,6 +75,8 @@ For adding support for other cloud providers, open an issue or propose a PR.
 
 ## Get started using Claudie
 
+To try Claudie you can follow this few steps or feel free to visit our [documentation](docs.claudie.io).
```

Comment:
There's a mistake: `these few steps`

And I'm not sure this is the best way or place to introduce the docs webpage

Reply:
I also wonder whether we should update all links to documentation to instead point to the webpage instead of github.
Thoughts?

Reply:
> I also wonder whether we should update all links to documentation to instead point to the webpage instead of github. Thoughts?

This is a good idea IMO.

> And I'm not sure this is the best way or place to introduce the docs webpage

I currently haven't found any better.